ortools/linear_solver/xpress_interface.cc

```diff
@@ -265,6 +274,9 @@ class XpressInterface : public MPSolverInterface {
   unique_ptr<int[]> mutable mCstat;
   unique_ptr<int[]> mutable mRstat;
 
+  const int* initCstat;
+  const int* initRstat;
+
```

Comment:
These are temporary objects, I'm not sure we need to duplicate them. We might just re-use them

Reply:
If not, `clear()` them after using them